### DISCUSSION TITLE
Parameter which removes not required classes in Annotations

### DISCUSSION THREAD

**Discussion by PrashantDixit0** [obss/sahi#906] [Ideas]
Adding a feature for keeping only required classes in annotation and removing other classes annotations for better comparison with other models

**Comment by jokober** [comment]
Not sure if this answers your questions but sahi provides some tools to handle coco datasets. You can also filter by categories and change the category ids. It is described here: https://github.com/obss/sahi/blob/main/docs/coco.md
